README.md

```diff
@@ -34,6 +36,8 @@ You'll need to make an account (free) [here](https://app.platerecognizer.com/acc
 
 You can specify a custom url for the plate_recognizer api by adding `api_url` to your config:
 
+If you're using CodeProject.AI, you'll need to comment out plate_recognizer as shown above and update "api_url" with your CodeProject.AI Service API URL.
```

Comment:
Two things. 

- The example code here goes with the sentence above. 

- Instead of commenting out the original example code at the top add the partial config here for your new code